### PR TITLE
add 'AfterStep hook' to ignored step names

### PR DESCRIPTION
**Pull request by irodkin**
I don't know about other types of hooks that are not ignored by formatter yet but they may exist.

### CODE REVIEW

File: lib/allure-cucumber/formatter.rb [hunk 1]
```diff
@@ -122,7 +122,8 @@ def after_test_step(test_step, result)
         if (!@before_hook_exception) && result.methods.include?(:exception)
           @before_hook_exception = result.exception
         end
-      elsif test_step.name != 'After hook'
+      end
```
Comment (mikesalvia) [inline lib/allure-cucumber/formatter.rb]: Should leave this as `elsif` why would it be changed?

File: lib/allure-cucumber/formatter.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@ class Formatter
 
     include AllureCucumber::DSL
     
-    TEST_HOOK_NAMES_TO_IGNORE = ['Before hook', 'After hook']
+    TEST_HOOK_NAMES_TO_IGNORE = ['Before hook', 'After hook', 'AfterStep hook']
```
Comment (mikesalvia) [inline lib/allure-cucumber/formatter.rb]: I was thinking about starting to use AfterStep so this is welcome. I think there is a BeforeStep hook as well not positive.

Comment (mikesalvia) [reply lib/allure-cucumber/formatter.rb]: There is no before the hook. @baev If this PR/branch is stale I am going to create a PR for this.

Comment (baev) [reply lib/allure-cucumber/formatter.rb]: sounds good to me

**Comment by KorranForce** [comment]
Cucumber now have new formatter API (and I rewrote allure-cucumber based on it long time ago) so I don't have interest in supporting old one - sorry. You can do whatever you want with this PR and code in it.

**Comment by baev** [comment]
@KorranForce can you send the pull request with your changes here?

**Comment by baev** [comment]
I'd like to keep the repo updated but I'm not familiar with Cucumber and Ruby, and our team has no Ruby dev at the moment, so it would be nice if you guys help us to keep the project updated.

**Comment by KorranForce** [comment]
My changes are based on customized and slightly outdated version of allure-ruby-adaptor-api lib (which is now called allure-framework/allure-ruby-commons, I guess). So PR is somewhat complicated. If it would be useful here are the links to versions that are being used in our autotests environment for nearly a year or so:
https://github.com/KorranForce/allure-ruby-adaptor-api/tree/develop
https://github.com/KorranForce/allure-cucumber/tree/develop

**Comment by baev** [comment]
this repository is deprecated, the new version available here https://github.com/allure-framework/allure-ruby

**Pull request closed without merging**